spec/app/app_spec.rb

```diff
@@ -54,7 +52,8 @@
 
     it "should let the user know who won" do
       game = GameStateManager.new(Human.new("X", true), Human.new("Y"))
-      game.store = [["X", "X", "X", 4, 5, 6, 7, 8, 9]]
+      game.store = [["X", "X", 3, 4, 5, 6, 7, 8, 9]]
```

Comment:
why is this nested inside an empty array?

Reply:
Store is an array of board states which are arrays

Reply:
what use case do you need multiple board states?

Reply:
The game_state object needs to keep track of all the game states up to that point in order to go "back in time" for the undo function. If there is only one game state at a time, there is no way to know what moves were made.

Reply:
gotcha, thanks for the quick answer.